lib/index.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2021 Edge Network Technologies Limited
+// Use of this source code is governed by a GNU GPL-style license
+// that can be found in the LICENSE.md file. All rights reserved.
+
+import Dotenv from 'dotenv'
+
+// Load environment variables from .env file if present
+Dotenv.config()
+
+// Provide helper methods to load environment variables
+export class Config {
+  protected static getEnvArray(key: string, defaultValue: string[]): string[] {
+    const value = process.env[key]
+    return value ? value.split(',') : defaultValue
```

Comment:
An input to specify the delimiter would probably be worthwhile here, perhaps as optional arg3. There may be cases where comma is not appropriate or conventional, e.g. it's more usual to supply an array of paths with `/colon:/delimiters:/like/this`

Reply:
Great idea!